journal-letters/modernized/10-chapter-9.adoc

```diff
@@ -347,7 +363,7 @@ There is great need of care in this respect,
 as well as not to exceed the bounds which Truth prescribes.
 May Divine goodness open our understandings,
 and more and more replenish our minds with that faith, which overcomes slavish fear,
-and gives the victory, 9th.--Was at a marriage, where I was silent,
+and gives the victory. 9th.--Was at a marriage, where I was silent,
```

Comment:
maybe go ahead and bump the part after `victory.` to the next line, since you corrected it to be a new sentence.